src/Cake.Curl.sln

```diff
@@ -15,33 +15,36 @@ Global
-		{2A32E458-9FB8-4294-B88E-6432C908C915}.Debug|x86.Build.0 = Debug|x86
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Debug|x64.Build.0 = Debug|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Debug|x86.Build.0 = Debug|Any CPU
 		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|Any CPU.Build.0 = Release|Any CPU
-		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x64.ActiveCfg = Release|x64
-		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x64.Build.0 = Release|x64
-		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x86.ActiveCfg = Release|x86
-		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x86.Build.0 = Release|x86
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x64.ActiveCfg = Release|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x64.Build.0 = Release|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x86.ActiveCfg = Release|Any CPU
+		{2A32E458-9FB8-4294-B88E-6432C908C915}.Release|x86.Build.0 = Release|Any CPU
 		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x64.ActiveCfg = Debug|x64
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x64.Build.0 = Debug|x64
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x86.ActiveCfg = Debug|x86
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x86.Build.0 = Debug|x86
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x64.Build.0 = Debug|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Debug|x86.Build.0 = Debug|Any CPU
 		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|Any CPU.Build.0 = Release|Any CPU
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x64.ActiveCfg = Release|x64
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x64.Build.0 = Release|x64
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x86.ActiveCfg = Release|x86
-		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x86.Build.0 = Release|x86
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x64.ActiveCfg = Release|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x64.Build.0 = Release|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x86.ActiveCfg = Release|Any CPU
+		{CFE6D136-69E4-4C49-A10E-CE0328445926}.Release|x86.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {7AD75399-5866-4EF0-A49F-0D52DDE153D6}
```

Comment:
No need to modify this file.

Reply:
@ecampidoglio It seems that we hit [this issue](https://github.com/dotnet/project-system/issues/1821). I can revert the changes but every time I open the solution, Visual Studio makes a change. I use latest Visual Studio 2017 Community Edition on Windows. Do you have problem with updated project GUIDs? What IDE/platform do you use?

Reply:
Interesting. I use Rider on macOS or just Vim in the terminal.

I think we should revert the change for the time and being and watch out for it in the future. You could also tell Git to [ignore any changes to the solution file](https://git-scm.com/docs/git-update-index#_skip_worktree_bit), as it's unlikely that we'll be adding a new project anytime soon:

```sh
git update-index --skip-worktree src/Cake.Curl.sln
```

Reply:
@ecampidoglio Thanks for the git tip!